src/noveum_trace/integrations/livekit/livekit.py

```diff
@@ -0,0 +1,1087 @@
+"""
+LiveKit STT/TTS integration for noveum-trace.
+
+This module provides wrapper classes that automatically trace LiveKit STT and TTS
+operations, capturing audio files and metadata as span attributes.
+"""
+
+from __future__ import annotations
+
+import logging
+from pathlib import Path
+from typing import Any, Optional, Union
+
+from noveum_trace.core.context import get_current_trace
+from noveum_trace.core.span import SpanStatus
+from noveum_trace.integrations.livekit.livekit_constants import (
+    STT_CONFIDENCE_DEFAULT_VALUE,
+    STT_END_TIME_DEFAULT_VALUE,
+    STT_IS_PRIMARY_SPEAKER_DEFAULT_VALUE,
+    STT_LANGUAGE_DEFAULT_VALUE,
+    STT_SPEAKER_ID_DEFAULT_VALUE,
+    STT_START_TIME_DEFAULT_VALUE,
+    STT_TRANSCRIPT_DEFAULT_VALUE,
+    TTS_DELTA_TEXT_DEFAULT_VALUE,
+    TTS_INPUT_TEXT_DEFAULT_VALUE,
+    TTS_NUM_CHANNELS_DEFAULT_VALUE,
+    TTS_REQUEST_ID_DEFAULT_VALUE,
+    TTS_SAMPLE_RATE_DEFAULT_VALUE,
+    TTS_SEGMENT_ID_DEFAULT_VALUE,
```

Comment:
Please verify that we save these in each span as well and other configs if they are there. We need to be able to save every config.